### PR TITLE
DOCSP-43423 Add limit returned results page

### PR DESCRIPTION
**Pull request by stephmarie17**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-rust/blob/master/REVIEWING.md)

JIRA - <https://jira.mongodb.org/browse/DOCSP-43423>
Staging - <https://deploy-preview-134--docs-rust.netlify.app/fundamentals/crud/read-operations/limit/>

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [x] Did you run a grammar-check?
- [x] Are all the links working?
- [x] Are the [facets and meta keywords](https://wiki.corp.mongodb.com/display/DE/Docs+Taxonomy) accurate?

### CODE REVIEW

File: source/fundamentals/crud/read-operations/limit.txt [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+---------------
+
+You can specify the maximum number of documents to return in a query or in an
+aggregation pipeline.
+
+This section describes how to limit results in the following ways:
+
+- :ref:`limit() method <rust-limit-method>`: Chain the ``limit()`` method to the
+  ``find()`` method
+- :ref:`FindOptions struct <rust-findoptions-limit>`: Use the ``limit()`` option
```
Comment (norareidy) [inline source/fundamentals/crud/read-operations/limit.txt:65]: S: since this limit() is the builder method that sets the limit option, I'd reword to "Use the limit option" or "Use the limit() builder method"
```suggestion
- :ref:`FindOptions struct <rust-findoptions-limit>`: Use the ``limit`` option
```

File: source/fundamentals/crud/read-operations/limit.txt [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+      Document({"_id": Int32(4), "name": String("A Dance with Dragons"), "author": String("Martin"), "length": Int32(1104)})
+
+Additional Information
+----------------------
+
+To learn more about the operations mentioned in this guide, see the following guides:
+
+- :ref:`rust-query-guide`
+- :ref:`rust-retrieve-guide`
+- :ref:`rust-compound-operations`
```
Comment (norareidy) [inline source/fundamentals/crud/read-operations/limit.txt:169]: S: I don't think you mention any of the compound operations on this page, so you can remove this link
```suggestion

```

File: source/fundamentals/crud/read-operations/limit.txt [hunk 3]
```diff
@@ -0,0 +1,184 @@
...
+
+To learn more about any of the methods or types discussed in this guide, see the
+following API documentation:
+
+- `find() <{+api+}/struct.Collection.html#method.find>`__
+- `FindOptions <{+api+}/options/struct.FindOptions.html>`__
+- `FindOneOptions <{+api+}/options/struct.FindOneOptions.html>`__
+- `Cursor <{+api+}/struct.Cursor.html>`__
+- `aggregate() <{+api+}/struct.Collection.html#method.aggregate>`__
+- `AggregateOptions <{+api+}/options/struct.AggregateOptions.html>`__
```
Comment (norareidy) [inline source/fundamentals/crud/read-operations/limit.txt:184]: S: I also think you can remove FindOneOptions and AggregateOptions as they aren't mentioned
```suggestion
- `find() <{+api+}/struct.Collection.html#method.find>`__
- `FindOptions <{+api+}/options/struct.FindOptions.html>`__
- `Cursor <{+api+}/struct.Cursor.html>`__
- `aggregate() <{+api+}/struct.Collection.html#method.aggregate>`__
```

File: source/includes/fundamentals/code-snippets/crud/limit.rs [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+
+    let mut cursor = my_coll.find(filter).with_options(find_options).await?;
+
+    while let Some(result) = cursor.try_next().await? {
+        println!("{:?}", result);
+    }
+// end-limit-options-example
+
+// Retrieves documents in the collection, sorts results by their "length" field
+// values, the limits the results to the first document.
```
Comment (norareidy) [inline source/includes/fundamentals/code-snippets/crud/limit.rs:84]: S:
```suggestion
// values, then limits the results to the first document.
```

**Review by norareidy — APPROVED** [review]
LGTM with some suggestions!

**Review by abr-egn — APPROVED** [review]
